### DISCUSSION TITLE
VPN Server Template

### DISCUSSION THREAD

**Discussion by jadonbrownza** [vitodeploy/vito#323] [Ideas]
Server template to install OpenVPN on a server.

It has a webpanel for administrators to manage credentials.

**Comment by jadonbrownza** [comment]
For reference:
https://www.youtube.com/watch?v=3F18KT8W7CQ

**Comment by saeedvaziry** [comment]
we had openVPN server type in the 0.x but removed it for now to focus more on the application. maybe in the future will consider it again
